lib/src/features/ipc/handlers/get_cookbooks_handler.dart

```diff
@@ -10,9 +12,10 @@ class GetCookbooksHandler implements IPCHandler{
     final defaultResponse = SDKIPCResponse<Cookbook>(success: response.success, action: response.action, data: Cookbook.create(), error: response.error, errorCode: response.errorCode);
     try {
       if (response.success) {
-        defaultResponse.data =  Cookbook.create()..mergeFromProto3Json(response);
+        defaultResponse.data =  Cookbook.create()..mergeFromProto3Json(jsonDecode(response.data));
       }
     } on  Exception catch (_){
```

Comment:
Is it possible to catch a more specific exception here?

Reply:
@kjawadDeveloper written codes.

Reply:
Yes